test/smoke/default/upgrade.rb

```diff
@@ -0,0 +1,6 @@
+# # encoding: utf-8
+
+# Inspec test for recipe osl-openstack::upgrade
+
+# The Inspec reference, with examples and extensive documentation, can be
```

Comment:
Empty smoke file?

Reply:
It's from the code_generator. We probably should fix this.